src/store/ChartState.js

```diff
@@ -72,6 +73,8 @@ class ChartState {
         showLastDigitStats = false,
         startEpoch,
         symbol,
+        crosshairState,
+        chartSize,
```

Comment:
@shayan-binary-pr The top widget also need to zoom the chart by setting the number of the bars(ticks) on the screen. since it doesn't know how much should it zoom to get the appropriate zoom level by invoking zoom in/out functions, it would be better to give the ability to set the number visible bars on the screen.

Reply:
as we talked, we are going to add new pop for setting `MaxTick`